package.yaml

```diff
@@ -29,6 +29,7 @@ dependencies:
 - text >= 1.2.0
 - time >= 1.8.0
 - unordered-containers >= 0.2.8
+- vector >= 0.12
```

Comment:
This can probably be lower, any suggestions?

Reply:
This covers Stackage LTS 12, which is for GHC 8.4, the latest GHC version this package is tested with, so it seems good to me.